### PR TITLE
View book entry details

### PR DESCRIPTION
**Pull request by i-zor066**
Added the detail views for search results and list results.

| Search results details | List entry details |
| --- | --- |
| search-result-details | list-details |

### CODE REVIEW

**Comment by Electryc** [comment]
I took the liberty to modify a bit your description :tongue: some table magic :sparkles:

File: MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/BookDetails.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.izor066.android.mediatracker.R;
+import com.izor066.android.mediatracker.api.model.Book;
+import com.squareup.picasso.Picasso;
+
+public class BookDetails extends AppCompatActivity implements View.OnClickListener {
+
+    String TAG = getClass().getSimpleName();
```
Comment (Electryc) [inline MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/BookDetails.java]: `private static final` :smile:

File: MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/BookDetails.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    Book book;
```
Comment (Electryc) [inline MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/BookDetails.java]: All of the below should be `private`.

File: MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/SearchResultDetails.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import android.widget.Toast;
+
+import com.izor066.android.mediatracker.MediaTrackerApplication;
+import com.izor066.android.mediatracker.R;
+import com.izor066.android.mediatracker.api.model.Book;
+import com.squareup.picasso.Picasso;
+
+public class SearchResultDetails extends AppCompatActivity implements View.OnClickListener {
+
+    String TAG = getClass().getSimpleName();
```
Comment (Electryc) [inline MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/SearchResultDetails.java]: `private static final` the same :smile:

File: MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/SearchResultDetails.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    Book book;
```
Comment (Electryc) [inline MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/SearchResultDetails.java]: The same here, all of the below should be `private`.

File: MediaTracker/app/src/main/res/layout/activity_book_details.xml [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+        <LinearLayout android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:id="@+id/viewOne"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_weight="0.6"
```
Comment (Electryc) [inline MediaTracker/app/src/main/res/layout/activity_book_details.xml]: For weights I usually use percentage based numbers, so 0.6 would be `60` and 0.4 `40`, but it's just a matter of preference, if you understand them better with 0.6 and 0.4 that's fine :+1:

File: MediaTracker/app/src/main/res/layout/activity_search_result_details.xml [hunk 6]
```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.izor066.android.mediatracker.ui.BookDetails">
+
+    <android.support.design.widget.CoordinatorLayout
```
Comment (Electryc) [inline MediaTracker/app/src/main/res/layout/activity_search_result_details.xml:9]: Uuu `CoordinatorLayout` :sparkles:

**Comment by i-zor066** [comment]
All the comments are implemented, the additions to the detailed view are functional:

List details:

list-details-entry

Search result, default values:

search-result-default-values

Search result:

search-details-001

**Comment by Electryc** [comment]
Looking good! :+1:

**Pull request merged**